static/json/languages/en_US.json

```diff
@@ -231,7 +231,7 @@
       "use_flashbots": "Use Flashbots",
       "clear_transactions_and_nonces": "Clear Pending Transactions",
       "clear_transactions_and_nonces_success": "Pending transactions cleared",
-      "flashbots_description": "Flashbots protects your transactions and reduces the likelihood of transaction failures. Learn more."
+      "flashbots_description": "Flashbots protects your transactions and reduces the likelihood of transaction failures. "
```

Comment:
The trailing space may get dropped when translating, so may need to handle the padding differently